src/components/App/App.tsx

```diff
@@ -12,27 +10,26 @@ import React, { FC } from "react";
 import "./App.css";
 
 const App: FC = () => {
-    const { isDarkMode, updateState, name, email, company, title, linkedin, phone } = useSignatureContext();
+    const { updateState, name, email, company, title, linkedin, phone } = useSignatureContext();
     const { profile } = useAuthenticationContext();
 
     return (
-        <ProviderV3 theme={isDarkMode ? darkTheme : lightTheme} colorScheme={"light"} height="100%">
```

Comment:
Moving the Spectrum provider one level up such that the routes that the router renders have the appropriate styling